CHANGELOG.md

```diff
@@ -17,7 +18,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
   - JDBCRecipientRewriteTable implementation. Please use JPARecipientRewriteTable instead.
   - JamesUsersJdbcRepository, DefaultUsersJdbcRepository. Please use JpaUsersRepository instead.
   - MailboxQuotaFixed matcher. Please use IsOverQuota instead.
-- UsersFileRepository, which was marked as deprecated for years
+  - UsersFileRepository, which was marked as deprecated for years
```

Comment:
no this was on purpose, it's not relate to JAMES-2703

Reply:
ah ok, i tought it was an alignment problem.